dlx_rest/api/utils.py

```diff
@@ -193,7 +193,7 @@ def brief_bib(record):
     if record.get_value('245', 'a'):
         head = ' '.join(record.get_values('245', 'a', 'b', 'c'))
     elif record.get_value('700', 'a') or record.get_value('710', 'a'):
-        head, member = record.get_value('700', 'a'), record.get_value('710', 'a')
+        head, member = ' '.join([record.get_value('700', 'a'), record.get_value('700', 'g') or None]), record.get_value('710', 'a')
```

Comment:
This doesn't seem to work in the dev database. Onscreen it says invalid search, but the app throws the following exception:

[2023-07-25 14:16:28,505] ERROR in app: Exception on /api/marc/bibs/records [GET]
Traceback (most recent call last):
  File "/home/aaron/dev/dlx-rest/venv/lib/python3.10/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/aaron/dev/dlx-rest/venv/lib/python3.10/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/aaron/dev/dlx-rest/venv/lib/python3.10/site-packages/flask_restx/api.py", line 404, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/aaron/dev/dlx-rest/venv/lib/python3.10/site-packages/flask/views.py", line 84, in view
    return current_app.ensure_sync(self.dispatch_request)(*args, **kwargs)
  File "/home/aaron/dev/dlx-rest/venv/lib/python3.10/site-packages/flask_restx/resource.py", line 46, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/aaron/dev/dlx-rest/dlx_rest/api/__init__.py", line 278, in get
    data = [make_brief(r) for r in recordset]
  File "/home/aaron/dev/dlx-rest/dlx_rest/api/__init__.py", line 278, in <listcomp>
    data = [make_brief(r) for r in recordset]
  File "/home/aaron/dev/dlx-rest/dlx_rest/api/utils.py", line 196, in brief_bib
    head, member = ' '.join([record.get_value('700', 'a'), record.get_value('700', 'g') or None]), record.get_value('710', 'a')
TypeError: sequence item 1: expected str instance, NoneType found